com.hazelcast.jet.core.OperationLossTest.when_connectionDroppedWithoutMemberLeaving_then_jobRestarts

_master_ (commit c9b227b736e7d864158b8804cfa614499dbefd40)

Failed on Sonar: http://jenkins.hazelcast.com/view/Jet/job/jet-oss-master-sonar-nightly/186/testReport/com.hazelcast.jet.core/OperationLossTest/when_connectionDroppedWithoutMemberLeaving_then_jobRestarts/

Stacktrace:
```
java.lang.AssertionError: expected:<2> but was:<1>
	at org.junit.Assert.fail(Assert.java:88)
	at org.junit.Assert.failNotEquals(Assert.java:834)
	at org.junit.Assert.assertEquals(Assert.java:645)
	at org.junit.Assert.assertEquals(Assert.java:631)
	at com.hazelcast.jet.core.OperationLossTest.when_connectionDroppedWithoutMemberLeaving_then_jobRestarts(OperationLossTest.java:177)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at com.hazelcast.test.FailOnTimeoutStatement$CallableStatement.call(FailOnTimeoutStatement.java:106)
	at com.hazelcast.test.FailOnTimeoutStatement$CallableStatement.call(FailOnTimeoutStatement.java:98)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.lang.Thread.run(Thread.java:748)
```

Standard output:
```
Hiccups measured while running test 'when_connectionDroppedWithoutMemberLeaving_then_jobRestarts(com.hazelcast.jet.core.OperationLossTest):'
13:05:40, accumulated pauses: 55 ms, max pause: 22 ms, pauses over 1000 ms: 0
```

--- a/hazelcast-jet-core/src/test/java/com/hazelcast/jet/core/OperationLossTest.java
+++ b/hazelcast-jet-core/src/test/java/com/hazelcast/jet/core/OperationLossTest.java
@@ -174,7 +174,7 @@ public class OperationLossTest extends JetTestSupport {
         dag.edge(between(source, sink).distributed());
         Job job = instance1.newJob(dag);
         assertJobStatusEventually(job, RUNNING);
-        assertEquals(2, NoOutputSourceP.initCount.get());
+        assertTrueEventually(() -> assertEquals(2, NoOutputSourceP.initCount.get()));
 
         Connection connection = Util.getMemberConnection(getNodeEngineImpl(instance1),
                 instance2.getHazelcastInstance().getCluster().getLocalMember().getAddress());

